app/views/claimToAdjustPoa/PoaAdjustedView.scala.html

```diff
@@ -28,7 +28,7 @@
         link: link,
 )
 
-@(isAgent: Boolean, poaTaxYear: TaxYear, poaTotalAmount: BigDecimal, showOverdueCharges: Boolean)(implicit messages: Messages, request: RequestHeader, user: auth.MtdItUser[_])
+@(isAgent: Boolean, poaTaxYear: TaxYear, poaTotalAmount: BigDecimal, showOverdueCharges: Boolean, yourSelfAssessmentChargesFS: Boolean)(implicit messages: Messages, request: RequestHeader, user: auth.MtdItUser[_])
```

Comment:
worth to create a separate viewModel for this + include this saChargesUrl field into a new model; we can create a tech debt for these changes